Dolapo47/Auto-Mart
# write helper function to trim user request

#### What does this PR do?
- A summary of what this task is about
this PR is created to trim user's request
#### Description of Task to be completed?
- A list of the major tasks you did to complete this feature/bug/chore
create a helper function to trim user's request
import function as a middleware to the routes
#### How should this be manually tested?
- Outlined steps on how someone else can successfully test this feature/chore/bug to ensure it works as expected
run npm start on the terminal
put in the user's input with space at both sides
the user's input is trimmed

#### What are the relevant pivotal tracker stories?
- PT Story link - (format - `[#167161023](https://www.pivotaltracker.com/n/projects/2346179)`)
